app/views/order_items/_order_item.html.erb

```diff
@@ -5,6 +5,17 @@
         <%= image_tag product.avatar %>
       </div>
       <div class="caption">
+        <h6>
+          <% if product.calories %>
+              <%= content_tag :small do
+                concat product.calories
```

Comment:
concat is for more complicated content_tag, just use `<%= content_tag :small, "#{product.calories} kcal" %>.
Or even better, move this to a helper function, as you're using it twice.